test/unit-test/gtclang/Parsing/TestParsing.cpp

```diff
@@ -0,0 +1,175 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
+//                         _       _
+//                        | |     | |
+//                    __ _| |_ ___| | __ _ _ __   __ _
+//                   / _` | __/ __| |/ _` | '_ \ / _` |
+//                  | (_| | || (__| | (_| | | | | (_| |
+//                   \__, |\__\___|_|\__,_|_| |_|\__, | - GridTools Clang DSL
+//                    __/ |                       __/ |
+//                   |___/                       |___/
+//
+//
+//  This file is distributed under the MIT License (MIT).
+//  See LICENSE.txt for details.
+//
+//===------------------------------------------------------------------------------------------===//
+
+#include "dawn/CodeGen/CXXUtil.h"
+#include "dawn/SIR/SIR.h"
+#include "gtclang/Support/ParsingHeader.h"
+#include "gtclang/Unittest/GTClang.h"
+#include "gtclang/Unittest/UnittestEnvironment.h"
+#include <fstream>
+#include <gtest/gtest.h>
+#include <map>
+
+using namespace dawn;
```

Comment:
dawn should always be explicitly qualified to avoid ADL problems (http://en.cppreference.com/w/cpp/language/adl)